include/ibamr/IBFESurfaceMethod.h

```diff
@@ -556,6 +624,8 @@ class IBFESurfaceMethod : public IBStrategy
     std::vector<libMesh::Order> d_default_quad_order;
     bool d_use_consistent_mass_matrix = true;
     bool d_use_direct_forcing = false;
+    double d_wss_calc_width, d_p_calc_width;
+    double d_traction_activation_time;
```

Comment:
Can we document somewhere what these values do? It's not immediately clear to me what these values do in the computations.

Should they also get default values?